### PR TITLE
Set GOMA options

### PR DESCRIPTION
**Pull request by tanin47**
- Set GOMA options.
- Ignore output because the output of compiling chrome is too large, and it slows down the process.

### CODE REVIEW

File: clusterfuzz/binary_providers.py [hunk 1]
```diff
@@ -286,6 +286,7 @@ def __init__(self, testcase_id, build_url, revision, current,
                                           current, goma_dir, source,
                                           binary_name, 'chromium_builder_asan')
     self.git_sha = sha_from_revision(self.revision, 'chromium/src')
+    self.name = 'chromium'
```
Comment (everestmz) [inline clusterfuzz/binary_providers.py]: Why do we need `self.name` here? I don't really see where it's used.

Comment (tanin47) [inline clusterfuzz/binary_providers.py]: It raised an exception because we used it when asking for source_directory.

Comment (everestmz) [inline clusterfuzz/binary_providers.py]: Right, forgot that.

**Comment by Dor1s** [comment]
Do you guys support building without goma? It feels that not everyone is always using it.

**Comment by tanin47** [comment]
Not at the moment. But that's a good point. We'll support it in the future.

**Pull request merged**